lib/reporter.js

```diff
@@ -57,6 +58,14 @@ let buildStackTraceJSON = function(error, projectRoot) {
   })
 }
 
+let normalizePath = path => {
+  return path.replace('file:///', '')                         // Randomly inserted file url protocols
+             .replace(/[/]/g, '\\')                           // Temp switch for Windows home matching
+             .replace(fs.getHomeDirectory(), '~')             // Remove users home dir for apm-dev'ed packages
```

Comment:
Missed this before it got merged, but it looks like instead of having to require fs, we can use the already-required `os` with `os.homedir()`?